src/code/z_parameter.c

```diff
@@ -4338,7 +4338,7 @@ void Interface_DrawClock(PlayState* play) {
     static s16 sThreeDayClockAlpha = 255;
     static s16 sClockAlphaTimer1 = 0;
     static s16 sClockAlphaTimer2 = 0;
-    static u16 sThreeDayClockHours[] = {
+    static u16 sThreeDayClockHours[26] = {
```

Comment:
```suggestion
    static u16 sThreeDayClockHours[] = {
```
We prefer not to have hard-constants in the array